CppCoreGuidelines.md

```diff
@@ -2846,16 +2846,29 @@ Not possible.
 
 Readability.
 Suppression of unused parameter warnings.
-If parameters are conditionally unused depending on the translation unit, declare them with the `[[maybe_unused]]` attribute.
 
 ##### Example
 
-    X* find(map<Blob>& m, const string& s, Hint);   // once upon a time, a hint was used
+    X* find(set<Blob>& s, const string& str, Hint);   // once upon a time, a hint was used
```

Comment:
suggest
`X* find(set<Blob>& ranges, const string& value, Hint);`
It's more readable,